pkg/gadgets/top/ebpf/types/types.go

```diff
@@ -102,6 +104,12 @@ func GetColumns() *columns.Columns[Stats] {
 	cols.MustSetExtractor("mapmemory", func(stats *Stats) (ret string) {
 		return fmt.Sprint(units.BytesSize(float64(stats.MapMemory)))
 	})
+	cols.MustSetExtractor("total_cpu", func(stats *Stats) (ret string) {
+		return fmt.Sprintf("%.4f%%", stats.TotalCpuUsage)
+	})
+	cols.MustSetExtractor("per_cpu", func(stats *Stats) (ret string) {
+		return fmt.Sprintf("%.4f%%", stats.PerCpuUsage)
+	})
```

Comment:
I think we can do the same by using `precision:4` in the column tag.

Reply:
ops, as noted by @flyth, the extractor is actually needed in this case to print the `%` sign. Sorry for the noise.